stackslib/src/chainstate/nakamoto/mod.rs

```diff
@@ -118,7 +148,10 @@ lazy_static! {
                      stacker_signature TEXT NOT NULL,
                      tx_merkle_root TEXT NOT NULL,
                      state_index_root TEXT NOT NULL,
-
+                     -- the consensus hash of the burnchain block that selected this block's tenure
+                     consensus_hash TEXT NOT NULL,
+                     -- the consensus hash of the burnchain block that selected this block's tenure
```

Comment:
Is this comment intended to be identical to the above field's comment?

Reply:
No -- will update